packages/db/src/runtime/index.ts

```diff
@@ -139,7 +138,7 @@ export function normalizeDatabaseUrl(envDbUrl: string | undefined, defaultDbUrl:
 		if (envDbUrl.startsWith('file://')) {
 			return envDbUrl;
 		}
-		return new URL(envDbUrl, pathToFileURL(process.cwd())).toString();
+		return new URL(envDbUrl, pathToFileURL(addTrailingSlash(process.cwd()))).href;
```

Comment:
Didn't you say you found out that `.href` excluded `file://` in some cases? According to https://developer.mozilla.org/en-US/docs/Web/API/URL/toString that should be the same...

Reply:
Ah yes, I also found that they are the same. The change to `.href` wa just left from testing and I can remove. However, that “add trailing slash” WAS needed for relative paths to resolve correctly. Added a unit test for that.

Reply:
Note that a previous PR of mine already added a trailing slash, though I didn't use the util

Reply:
@Princesseuh oh, I didn't see a fix in Astro DB for this. Was this somehow removed?

Reply:
I think your branch is just out of date 😄

Reply:
The PR in question: https://github.com/withastro/astro/pull/10925

Reply:
OH lol, you're right. I'll rebase